modules/outbound-domain-event-handler/src/shared/config.ts

```diff
@@ -43,6 +47,20 @@ const config = Convict({
         default: 'info',
         env: 'LOG_LEVEL',
     },
+    API_SERVER: {
+        ENABLED: {
+            doc: 'Whether to enable API server or not',
+            format: 'Boolean',
+            default: false,
+            env: 'API_SERVER_ENABLED',
+        },
+        PORT: {
+            doc: 'The port of the API server.',
+            format: 'port',
+            default: 8001,
+            env: 'API_SERVER_PORT',
```

Comment:
I think we need to make these specific to the handler now, perhaps something like this:

```suggestion
            env: 'DOMAIN_EVENT_API_SERVER_PORT',
```

Note we should go back and update the Command handler to match the same as above, except using something like a `COMMAND_EVENT_*` prefix